app/views/admin/hackathons/show.html.erb

```diff
@@ -14,19 +14,31 @@
     </turbo-frame>
 
     <% if @hackathon.logo.present? || @hackathon.banner.present? %>
-      <section style="display: flex;" class="pt3">
+      <section style="display: flex; gap: var(--spacing-2)" class="pt3">
         <% if @hackathon.logo.present? %>
-          <div>
-            <h2 class="heading--small">Logo</h2>
-            <%= image_tag @hackathon.logo, size: 128, style: "background-color: black;" %>
-          </div>
+          <turbo-frame id="logo">
```

Comment:
```suggestion
          <turbo-frame id="hackathon_logo">
```

Reply:
This is intentional because the generated ID for the `file_field` is `hackathon_logo`